nni/contrib/compression/base/apply_method.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
do we still need the folder "contrib"?

Reply:
we need `contrib` to hold some experimental feature or tutorial assets, but yes, compression v2.5 should be moved out after development is done.